Package@swift-6.0.swift

```diff
@@ -0,0 +1,134 @@
+// swift-tools-version: 6.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import CompilerPluginSupport
+import PackageDescription
+
+let package = Package(
+    name: "SafeDI",
+    platforms: [
+        .macOS(.v10_15),
+        .iOS(.v13),
+        .tvOS(.v13),
+        .watchOS(.v6),
+        .macCatalyst(.v13),
+        .visionOS(.v1),
+    ],
+    products: [
+        /// A library containing SafeDI macros, property wrappers, and types.
+        .library(
+            name: "SafeDI",
+            targets: ["SafeDI"]
+        ),
+        /// A SafeDI plugin that must be run on the root source module in a project.
+        .plugin(
+            name: "SafeDIGenerator",
+            targets: ["SafeDIGenerator"]
+        ),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/apple/swift-argument-parser.git", from: "1.2.0"),
+        .package(url: "https://github.com/apple/swift-collections.git", from: "1.0.0"),
+        .package(url: "https://github.com/apple/swift-syntax.git", from: "510.0.0"),
```

Comment:
We'll move this to 
```suggestion
        .package(url: "https://github.com/apple/swift-syntax.git", from: "600.0.0"),
```
once it is available

Reply:
```suggestion
        // TODO: Bump to 600.0.0 once it's available.
        .package(url: "https://github.com/apple/swift-syntax.git", from: "510.0.0"),
```